### DISCUSSION TITLE
Hexo配置中的URL

### DISCUSSION THREAD

**Discussion by lumit66** [hexojs/hexo#5244] [Question / Answer] [answered]
在URL配置中可以设置多个地址吗，如果可以又该怎么做呢？
![image](https://github.com/hexojs/hexo/assets/120006851/71b4161b-a38f-4ee8-aa11-2c378a25cb9f)

**Comment by stevenjoezhang** [comment, marked as answer]
不支持。你的使用场景是什么？
